_src/_data/contributors.yml

```diff
@@ -69,6 +69,12 @@
   role: PMC Member
   org: IBM
 
+- name: Guobao Li
+  apacheId: liguobao
+  githubId: EdgarLGB
+  role: Committer
+  Org: 
```

Comment:
Would you mind adding Criteo as my current org?

Reply:
As an excercise add this on yourself. with 
1. `git commit --amend `
2. `git commit --no-edit --date "now"`

So, this would be shown as j143 authored and EdgarLGB committed.

Reply:
ok